benches/histogram.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use test::Bencher;
+
+use prometheus::{HistogramOpts, Histogram, HistogramVec};
+
+#[bench]
+fn bench_histogram_with_label_values(b: &mut Bencher) {
+    let histogram = HistogramVec::new(HistogramOpts::new("benchmark_histogram",
+                                                         "A histogram to benchmark it."),
+                                      &["one", "two", "three"])
+        .unwrap();
+    b.iter(|| histogram.with_label_values(&["eins", "zwei", "drei"]).observe(3.1415))
+}
+
+#[bench]
+fn bench_histogram_no_labels(b: &mut Bencher) {
+    let histogram = Histogram::with_opts(HistogramOpts::new("benchmark_histogram",
+                                                            "A histogram to benchmark it."))
+        .unwrap();
+    b.iter(|| histogram.observe(3.1415))
```

Comment:
I think 3.1415 is ok. It is also 3.1415 in `prometheus/client_golang`, and we do not want the random generator to disturb our bench result.